catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/CategoryListFragment.java

```diff
@@ -741,9 +726,12 @@ private void initializeAdapter() {
 		return addHeader(toCategoryListItems(SENSORS_TOUCH, SENSORS_TOUCH_PARAMS), getString(R.string.formula_editor_device_touch_detection));
 	}
 
-	private List<CategoryListItem> getVisualSensorItems() {
-		return addHeader(toCategoryListItems(SENSORS_VISUAL_DETECTION, SENSORS_VISUAL_DETECTION_PARAMS),
-				getString(R.string.formula_editor_device_visual_detection));
+	private List<CategoryListItem> getFaceSensorItems() {
+		return addHeader(toCategoryListItems(SENSORS_FACE_DETECTION), getString(R.string.formula_editor_device_face_detection));
+	}
+
+	private List<CategoryListItem> getTextSensorItems() {
+		return addHeader(toCategoryListItems(SENSORS_TEXT_DETECTION), getString(R.string.formula_editor_device_text_detection));
```

Comment:
You should NOT just overwrite the changes with yours, they were added for a reason, please add the SENSORS_VISUAL_DETECTION_PARAMS also to these items accordingly(split them into 2 Lists).
This should also fix the Warning on Jenkins

Reply:
Ohh. Okay I'm sorry, will revert that.